ArrayTools/set.py

```diff
@@ -3,7 +3,7 @@
 class Set:
 
     def __init__(self, element_list):
-        self.elements = list()
```

Comment:
This is more Pythonic:
A sample class that takes an empty constructor:
```
class Category():
    
    def __init__(self, category="generic"):
        self.category = category
```
And here's using `for in range()` to create a list of a specific size and initialize each item to an instance of the class.
```
array_size = 8
my_array = list(Categories() for i in range(array_size))
print(my_array)
[<__main__.Category object at 0x106e4da90>, <__main__.Category object at 0x106e4df60>, <__main__.Category object at 0x106e5f630>, <__main__.Category object at 0x106e5f160>, <__main__.Category object at 0x106e5f828>, <__main__.Category object at 0x106e5f6a0>, <__main__.Category object at 0x106e5f710>, <__main__.Category object at 0x106e5f358>]
```
And if you still want to filter them, here's an example using a lambda:
```
sequence = ['g', 'e', 'e', 'j', 'k', 's', 'p', 'r'] 
filtered_list = list(filter(lambda e: e not in sequence, ['a', 'b', '3', 's', 'p', 'r']))
print(filtered_list)
['a', 'b', '3']
```